air.toml

```diff
@@ -0,0 +1,7 @@
+[format]
+line-width = 80
+indent-width = 2
+indent-style = "space"
+line-ending = "auto"
+persistent-line-breaks = true
+exclude = ["R/000-wrappers.R"]
```

Comment:
Doesn't the latest Air automatically skip formatting of auto-generated files?

Reply:
Oh yes indeed